tasks/extract_filepaths_for_granule/schemas/output.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "granules": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "granuleId": {
+              "type": "string"
+            },
+            "keys": {
+              "type": "array",
+              "items": [
+                {
+                  "type": "string"
```

Comment:
I don't see a lot of descriptions, and it's properties like this that make me want them. Recommend pulling descriptions from the main .py, then pointing the .py at these files as the authority on input/output/config.

Reply:
I agree, the output here should also be close to matching the input schema for reques_files.py https://github.com/nasa/cumulus-orca/blob/master/tasks/request_files/schemas/input.json

Reply:
added descriptions